.github/workflows/_test.yml

```
@@ -0,0 +1,177 @@
+      - name: Run tests
+        shell: bash
+        working-directory: ${{ inputs.test_working_dir }}
+        run: |
+          # Ensure all paths are relative to repo root
+            gotestsum \
+            --junitfile ${{ env.REPORTS_DIR }}/${{ inputs.test_name }}.xml \
+              --format testname \
+            --jsonfile ${{ env.REPORTS_DIR }}/${{ inputs.test_name }}.json \
+            -- \
+              -coverpkg=./pkg/... ./cmd/... -coverprofile=${{ env.COVERAGE_DIR }}/${{ inputs.test_name }}.coverage \
+                ${{ inputs.test_args }} \ 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)